sharktank/sharktank/types/tensors.py

```diff
@@ -378,6 +378,12 @@ def reshape(self, *args: Union[List[List[int]], List[int]]) -> "AnyTensor":
             shape = args[0]
         return reshape(self, shape)
 
+    def size(self, dim: Optional[int] = None) -> tuple[int]:
+        if dim is None:
+            return tuple(self.shape)
+        else:
```

Comment:
No `else` condition required. Just include the return when the condition is not taken.

Reply:
Done.